manifests/quick-start/sso/dex/dex-cm.yaml

```diff
@@ -15,7 +15,7 @@ data:
     staticClients:
       - id: argo-server
         redirectURIs:
-          - http://localhost:2746/oauth2/callback
+          - http://localhost:8080/oauth2/callback
```

Comment:
this is wrong, sorry, it should be 2746

Reply:
I know that UI runs on 8080

Reply:
If it is left to be 2746, the dex server returns an `Unregistered redirect_uri` error when I use SSO login with web UI.
![image](https://user-images.githubusercontent.com/2749804/128793336-a5cb0b3b-aeac-47e6-86fb-395507be2900.png)

Should I not update but add 8080 to URIs?

Reply:
fix it: https://github.com/argoproj/argo-workflows/pull/6503/commits/ad02c188e9a9479325e164bd2f04f701a589d595